cmd/mta.go

```diff
@@ -249,7 +249,7 @@ var updateParametersCmd = &cobra.Command{
 	Long:  "Update parameters",
 	Args:  cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return mta.RunModifyAndWriteHash("update parameters", updateParametersCmdPath, false, func() ([]string, error) {
+		return mta.RunModifyAndWriteHash("update parameters", updateParametersCmdPath, true, func() ([]string, error) {
```

Comment:
This should not be true, we want the user to first read the parameters and only then update. Force means you don't require the user to read the parameters before updating them.